net.eudic.dict.metainfo.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version='1.0' encoding='utf-8'?>
+<component type="desktop-application">
+  <!--Created with jdAppStreamEdit 9.2-->
+  <id>net.eudic.dict</id>
+  <name>Eudic</name>
+  <name xml:lang="zh">欧路词典</name>
+  <summary>An authoritative English dictionary software. A must-have tool for English learners.</summary>
+  <summary xml:lang="zh">权威的英语词典软件 英语学习者必备的工具</summary>
+  <developer>
+    <name>QianYan</name>
+    <name xml:lang="zh">上海倩言网络科技有限公司</name>
+  </developer>
+  <metadata_license>CC0-1.0</metadata_license>
+  <description>
+    <p>Software easy to use offline dictionary</p>
```

Comment:
can you translate or add more to the English description?

Reply:
added.